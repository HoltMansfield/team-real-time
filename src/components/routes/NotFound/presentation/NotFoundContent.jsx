import React from 'react'

export function NotFoundContent() {
  return (
    <div>
      404 - this page is not found
    </div>
  )
}
