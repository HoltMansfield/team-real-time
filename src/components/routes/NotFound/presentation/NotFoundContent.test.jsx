import React from 'react'
import renderer from 'react-test-renderer'
import { NotFoundContent } from './NotFoundContent'

it('renders correctly', () => {
  const component = renderer
    .create(<NotFoundContent />)
    .toJSON()
  expect(component).toMatchSnapshot()
})
