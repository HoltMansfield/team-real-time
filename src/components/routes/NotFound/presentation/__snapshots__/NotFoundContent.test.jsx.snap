// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  404 - this page is not found Do we have a design for this?
</div>
`;
